psiturk/experiment_errors.py

```diff
@@ -165,9 +166,10 @@ def to_dict(self):
         rv['message'] = self.message
         return rv
 
+
 class InvalidUsageError(ExperimentApiError):
     
     def __init__(self, *args, **kwargs):
         if 'status_code' not in kwargs:
             kwargs['status_code'] = 400
-        super(InvalidUsage, self).__init__(*args, **kwargs)
+        super(InvalidUsageError, self).__init__(*args, **kwargs)
```

Comment:
or just `super().__init__(*args, **kwargs)`

Reply:
:+1: that was a holdover from needing to support python2 iirc